src/Components/Topics/Paper.jsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from "react";
+import { api } from "../../api";
+import Loading from "../Loading";
+import Error from "../Error";
+
+const Paper = () => {
+  const [paperArticles, setPaperArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+
+  useEffect(() => {
+    try {
+      const loadArticles = async () => {
+        const data = await api.articles.get();
+        const paperArticles = data.filter(
+          (article) => article.topic === "paper"
```

Comment:
We want to add a topic query ?topic="" onto the api call so we get all articles with that topic back